js/src/elements/form-behavior.ts

```diff
@@ -26,6 +27,7 @@ export class FormBehaviorElement extends HTMLElement {
 
 	validators: Validator[] = [
 		this.validateMinLength.bind(this), // min length before required
+		// this.validateGroupRequired.bind(this), // Checkbox or Radio groups
```

Comment:
Should this be commented out? If we need to get rid of it, let's delete this line of code altogether.

Reply:
Removed